python/deltalake/table.py

```diff
@@ -456,6 +456,27 @@ def vacuum(
             enforce_retention_duration,
         )
 
+    def update(
+        self,
+        updates: Dict[str, str],
+        predicate: Optional[str] = None,
+        writer_properties: Optional[Dict[str, int]] = None,
+        strict_cast: bool = True,
```

Comment:
Maybe a better name is `error_on_type_mismatch: boal = True`? As a reader of this, I'm not sure what updating values has to do with casting.

Reply:
Good one, that would be clear enough!